infrastructure/ansible/roles/postgres/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+- name: Ensure latest bash and ssl
```

Comment:
These tasks were in 'lobby_database', they are still specific to database. The other parts of 'lobby-database' tasks have been moved to 'flyway' role. The two sets of tasks are split based on what is specific to application vs generic to setting up the postgres database. So for example, if we were to deploy multiple schemas, or deploy multiple postgreses, we could just add more 'flyway'-like roles or re-use this postgres role.